prep-css-flexbox/align-items.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - align-items</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="starwars.css" />
+    <style>
+      .card-container {
+        display: flex;
+        flex-basis: calc(100% / 3);
+        min-height: 100vh;
+        align-items: center;
+        justify-content: center;
+      }
```

Comment:
Hi Paul, in these challenges, you'll need to target two different elements. The card deck can be targeted with a `#card-deck` id selector

```css
#card-deck {

}
```

and the card container(s) can be targeted with a class selector.

```css
.card-container {

}
```

So for each of your HTML pages, you will need two rule sets, one for `#card-deck` and one for `.card-container`.

Some of the properties for flexbox go on the flex container (in this case the `#card-deck`) such as `display: flex` or `align-items`. Others belong on the `.card-container`, such as `flex-basis`.